colossalai/tensor/colo_tensor.py

```diff
@@ -39,7 +38,7 @@ def __init__(
         self._shard_spec = shard_spec
 
     @property
-    def shard_spec(self) -> Optional[str]:
+    def shard_spec(self) -> Optional[TensorSpec]:
```

Comment:
If the default is a DP compute pattern. the spec won't be None.

Reply:
Done